content/kapp-controller/docs/latest/packaging-workflows.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Packaging Workflows
+---
+
+We recommend reading through the docs about the new [packaging
+apis](packaging.md) to familiarize yourself with the yaml used in these
+workflows.
+
+## Package Authoring
+
+## Package Consumption
+
+This workflow walks through the example contained in
+the [`packaging-demo`](https://github.com/vmware-tanzu/carvel-kapp-controller/tree/dev-packaging/examples/packaging-demo)
+directory and assumes all commands are run from within the directory.
+
+### Setup
+
+First, some [setup of rbac](security-model.md) is necessary and can be done using:
+
+```
+$ kapp deploy -a default-ns-rbac -f https://raw.githubusercontent.com/vmware-tanzu/carvel-kapp-controller/develop/examples/rbac/default-ns.yml
+```
+
+### Adding the Repo
+
+Now that we are setup, we need to make some packages available to the cluster.
+To do this, we can apply the contents of `repo.yml`, which contains a
+PackageRepository CR:
+
+```
+$ kapp deploy -a repo -f repo.yml
```

Comment:
Maybe we should only use urls for `-f` so the user doesn't need to download or manually set up anything?

Reply:
I was also going to put the actual yaml in to the steps with a little bit of a description of what it contains, so users could just copy and paste? The raw urls might be a lot to look at.

Reply:
Oh, I mean strictly adding so the user can run the command without downloading the file or setting it up locally. So the user should never look at the contents of the url. It's just to save them the step of doing anything locally. 

I think doing this along with describing the YAML makes sense. WDYT?